R/init.R

```diff
@@ -532,7 +532,7 @@ if (interactive() &&
       environment()
     })
 
-    if (!isFALSE(getOption("vsc.helpPanel", "Two"))) {
+    if (getOption("vsc.helpPanel", "Two") != FALSE) {
```

Comment:
We better use `!identical(getOption("vsc.helpPanel", "Two"), FALSE)` here to avoid being stopped by invalid values. Also consistent with other existing ones.